drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-backend/src/main/java/org/drools/workbench/screens/scenariosimulation/backend/server/util/ScenarioSimulationBuilder.java

```diff
@@ -41,4 +43,26 @@ public Simulation createSimulation(Path context, ScenarioSimulationModel.Type ty
                 return null;
         }
     }
+
+    public Background createBackground(Path context, ScenarioSimulationModel.Type type, String value) throws Exception {
+        switch (type) {
+            case RULE:
+                return ruleSimulationCreationStrategy.createBackground(context, value);
+            case DMN:
+                return dmnSimulationCreationStrategy.createBackground(context, value);
+            default:
+                return null;
```

Comment:
Question: can we throw an exception here? Like IlligalStateException? So to have a chance to understand what happened by logs in runtime?